controllers/metal3machine_controller_test.go

```diff
@@ -74,7 +74,6 @@ func setReconcileNormalExpectations(ctrl *gomock.Controller,
 	if tc.BootstrapNotReady {
 		m.EXPECT().HasAnnotation().MaxTimes(0)
 		m.EXPECT().GetBaremetalHostID(context.TODO()).MaxTimes(0)
-		m.EXPECT().Update(context.TODO()).MaxTimes(0)
```

Comment:
why removing it from here ? if bootstrap is not ready, Update should not be called (neither Associate btw).

Reply:
Fixed